Source/WebKit/GPUProcess/GPUConnectionToWebProcess.h

```diff
@@ -138,7 +138,7 @@ class GPUConnectionToWebProcess
     virtual ~GPUConnectionToWebProcess();
 
     bool isWebGPUEnabled() const { return m_preferences.isWebGPUEnabled; }
-    bool isWebGLEnabled() const { return m_preferences.isWebGLEnabled; }
+    bool isWebGLInGPUProcessEnabled() const { return m_preferences.isWebGLInGPUProcessEnabled; }
```

Comment:
I'm all for consistency, but I think it doesn't apply in this case:
- `isWebGPUEnabled()` and `isWebXREnabled()` are single boolean values reflecting the same flags in other processes.
- `isWebGLInGPUProcessEnabled()` is different, in that it combines the "traditional" app-wide `isWebGLEnabled()` with an additional flag allowing (or not) WebGL to run in the GPU process. I think the distinction here is important, and useful to highlight wherever it is used.
Please let me know if you can think of a better wording. `isGPUWebGLEnabled()`? `isWebGLEnabledAndUsesGPUProcess()`?

An alternative would be to have two separate & consistent flags `isWebGLEnabled()` and `useGPUProcessForWebGLEnabled()`, and check `isWebGLEnabled() && useGPUProcessForWebGLEnabled()` before handling GL messages from the GPU process.

Reply:
On reflection, I suppose that as you say we could interpret `GPUProcessPreferencesForWebProcess` members to be implicitly conditional upon each being allowed in the GPU process.
So `GPUProcessPreferencesForWebProcess::isWebGLEnabled()` means WebGL is currently enabled *and* it can run in the GPU process, etc.

So I'd be fine with reverting my name change; I just thought it was useful to add the clarification wherever it's used, to distinguish it at a glance from `WebPreferences::webGLEnabled()` (which does *not* include `useGPUProcessForWebGLEnabled`).

Reply:
My other question disappeared, which was: When is useGPUProcessForWebGLEnabled ever false?

I don't think we support WebGL *not* in the GPU process. OpenGL is definitely blocked in WebContent and so might Metal. Do we support WebGL in WK1? (These are questions for @kkinnunen-apple)

Reply:
Alright, I've undone the renaming, it makes for a much simpler patch! 😄

Reply:
> When is useGPUProcessForWebGLEnabled ever false?

If it's never going to be false, should it just be removed to avoid confusion? 🤷

But in UnifiedWebPreferences.yaml, I see:
```
UseGPUProcessForWebGLEnabled: [...]
  defaultValue: [...]
    WebKit:
      "ENABLE(GPU_PROCESS_BY_DEFAULT) && ENABLE(GPU_PROCESS_WEBGL_BY_DEFAULT)": true
      "USE(GRAPHICS_LAYER_WC)": true
      default: false
```
So it could theoretically be false if some combination of these all-caps flags are falsy.

Anyway, in the meantime this flag exists, so just in case it *could* be set to false somehow, we may as well respect it where appropriate.